### DISCUSSION TITLE
Have 'pdm show' command default to installed packages

### DISCUSSION THREAD

**Discussion by justasojourner** [pdm-project/pdm#619] [Ideas]
This relates to my other discussion topic https://github.com/pdm-project/pdm/discussions/618 because my project 'DRP' was made a package.

When I ran the `pdm show` command on the (newly created) package...

```
drp % pdm show drp
Name:                  Drp
Latest version:        0.1.2
Latest stable version: 0.1.2
Installed version:     0.0.0 <----- AND SHOWS THIS
Summary:               Drp is a Dropbox command line utility to ease upload, download and sharing of files
Author:                Sourabh Verma
Author email:          sourabh.coder@gmail.com
License:               GPLv3
Homepage:              https://github.com/sourabhv/drp
Project URLs:
Platform:              UNKNOWN
Keywords:              dropbox cli tool

```

I saw that it was referring to a package on PyPi that had the same name as my local package (project). Which made me think that it would be best for the command to, by default, refer to **local and installed PyPi packages**. 

Also note the command `pdm show drp` showed the **installed version** as being my (unwanted) package in conflict with the PyPi version. This could cause serious issues - what would happen if I did a `pdm update`?

There perhaps needs to be a default local 'namespace', and operations are done in that scope unless explicitly instructed otherwise?

**Comment by justasojourner** [comment]
Hi - just checking, any thoughts on the suggestion?

**Reply by frostming** [reply]
Type in `pdm show` without package name
